packages/markdown-docx/test/CiceroMarkToOOXMLRoundTrip.test.js

```diff
@@ -25,8 +25,11 @@ const { checkRoundTripEquality } = require('./helper');
 describe('Perform roundtripping between CiceroMark and OOXML', () => {
     const fileNames = fs.readdirSync(directoryName);
     for (const file of fileNames) {
-        it(`should parse ${file.replace('.json', '')}.`, async () => {
-            await checkRoundTripEquality(`${directoryName}/${file}`);
-        });
+        // step by step increase to avoid faliure of tests
+        if(file === 'text-and-emphasis.json'){
+            it(`should parse ${file.replace('.json', '')}.`, async () => {
```

Comment:
Skip tests using [`xit`](https://stackoverflow.com/a/32724129/11751642). Or mocha also has `skip()` API. Just check it out.

Reply:
@algomaster99 
I have tested it for the current test(text-and-emphasis.json) which the earlier transformer does and it passed. 
Also, for doing nesting, it would require addition of some more code so I will push the corresponding changes.
The changes will be as
1. `heading transformer`
2. `variable transformer`
3. `softbreak transformer`

These changes will ensure that we have completed the rewriting of transformer.

Reply:
You can do the rewriting in one PR. Just make sure the existing tests pass and add more as well. 

If rewriting somehow is not able to pass the existing test, you may skip it and deal with it in other PR.

Reply:
Yeah I intend to do so.